bin/oci-iscsi-config

```diff
@@ -1,1189 +1,34 @@
-#!/usr/bin/env python2.7
+#!/bin/sh
 
-# oci-utils
-#
 # Copyright (c) 2017, 2019 Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at
 # http://oss.oracle.com/licenses/upl.
 
-"""
-This utility assists with configuring iscsi storage on Oracle Cloud
-Infrastructure instances.  See the manual page for more information.
-"""
 
-import argparse
-import logging
-import os
-import subprocess
-import sys
-from datetime import timedelta
+# This utility assists with configuring iscsi storage on Oracle Cloud
+# Infrastructure instances.  See the manual page for more information.
 
-import oci_utils.oci_api
-from oci_utils import __ignore_file, iscsiadm, lsblk
-from oci_utils import _configuration as OCIUtilsConfiguration
-from oci_utils import (OCI_ATTACHMENT_STATE, OCI_VOLUME_SIZE_FMT)
-from oci_utils.cache import load_cache, write_cache
-from oci_utils.exceptions import OCISDKError
-from oci_utils.metadata import InstanceMetadata
+_CUT=/usr/bin/cut
+_PY=/usr/bin/python
+_PY2=/usr/bin/python2
+_PY3=/usr/bin/python3
 
-_user_euid = None
+. /etc/os-release
```

Comment:
This is not ideal, I would rather see we look to see if python3 is available and use it otherwise fall back and try to use python2

Reply:
now I test existence of python* binaries